src/connectReactGridLayoutBuilder.js

```diff
@@ -140,4 +130,4 @@ var connectReactGridLayoutBuilder = ReactGridLayout => class extends Component {
     );
   }
 };
-export default connectReactGridLayoutBuilder;
```

Comment:
Me, this line shouldn't be removed, but I can restore it from GitHub

Reply:
Thank you fo commitin those fixes!